store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10104,4 +10104,9 @@ TODO: delete them permanently from here
     indexing         - Generate index
   </desc>
 </attr>
+
+<attr id="9030" name="zimbraCountObjectsEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="11.0.0">
```

Comment:
can't we implement some type of caching mechanism on the server side instead of disabling the feature?

Reply:
Disabling through ldap attr was already discussed and part of ticket implementation. So, its implemented that way.

Reply:
The ability to disable the counting is necessary to avoid high load on ldap. Even if cache mechanism is introduced, the high load will happen at the first searching.
The cache mechanism should be implemented in a separate ticket as necessary. Having a cache might cause another issue like returning unexpected results due to old cache and/or increasing memory usage on a mailbox server.